android/src/main/java/com/mersive/capacitor/syscpu/SysCpuPlugin.java

```diff
@@ -26,4 +28,30 @@ public void getCpuInfo(PluginCall call) {
         ret.put("load", implementation.getCpuLoad());
         call.resolve(ret);
     }
+
+    @PluginMethod
+    public void getZoneTemp(PluginCall call) {
+        JSObject ret = new JSObject();
+        ret.put("temp", getZoneTemp(call.getInt("zone")));
+        call.resolve(ret);
+    }
+
+    private Double getZoneTemp(int zone) {
+        Process process;
+        try {
+            process = Runtime.getRuntime().exec("cat sys/class/thermal/thermal_zone" + (zone + 2) + "/temp");
```

Comment:
I'm surprised you don't need the `/` in front of the path. Are you guaranteed to always run this from the root directory?

Reply:
I didn't even notice that. I just copy pasted the command from Solstice. I'll add in the slash just to be safe but, yeah it seems to work without it.